content/ui-ux.md

```diff
@@ -0,0 +1,514 @@
+---
+title: User Interfaces and User Experience
+---
+
+After reading this guide, you'll know:
+
+1. How to build re-usable client side components in any templating language
+2. How to build a styleguide to allow you to visually test such pure components
+3. Patterns for building front end components in a performant way in Meteor
+4. How to design responsively across device sizes, accessibly for different users, and universally across languages.
+5. How to build components that can cope with a variety of different data sources
+6. How to use animation to keep users informed of changes
+
+## Reusable components
```

Comment:
Maybe "Your UI is built out of components" or just "UI components"? I feel like the word "reusable" implies pure, so it's a bit confusing for a first heading.